ores/applications/wsgi.py

```diff
@@ -0,0 +1,56 @@
+"""
```

Comment:
This seems to have a lot of the same code as `celery.py`. Could we pull out the similarities?